experimental/apps-mcp/lib/providers/databricks/sql.go

```diff
@@ -0,0 +1,67 @@
+package databricks
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/databricks/databricks-sdk-go/service/sql"
+)
+
+// ExecuteQueryArgs represents arguments for executing a SQL query
+type ExecuteQueryArgs struct {
+	Query string `json:"query"`
+}
+
+// ExecuteQuery executes a SQL query and returns the results
+func (c *Client) ExecuteQuery(ctx context.Context, query string) ([]map[string]any, error) {
```

Comment:
This helper could be something to retain as CLI command as long as https://github.com/databricks/cli/pull/3905 is not merged (or in case it's better than the command from that PR).